custom_components/alexa_media/notify.py

```
@@ -214,19 +214,29 @@ async def async_send_message(self, message="", **kwargs):
-                _LOGGER.debug("Processed Target by json: %s", processed_targets)
-            except json.JSONDecodeError:
-                if "," in target:
-                    processed_targets += [
-                        item.strip() for item in target.split(",") if item.strip()
-                    ]
-                else:
-                    processed_targets.append(target.strip())
-                _LOGGER.debug("Processed Target by string: %s", processed_targets)
+
+
+for target in targets:
```

Pylint (4.0.5)

[error] Using variable 'targets' before assignment

(E0601)